src/Components/Header/header.jsx

```diff
@@ -1,10 +1,12 @@
 import React from "react";
 
-import logo from "./2222.jpg";
+import s from './header.module.css';
```

Comment:
Consider using "styles" instead of "s" - it is more readable.

Reply:
i changed all "s" variable to "style"